final_task/rss_reader/topdf.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from xhtml2pdf import pisa
+import cgi
+import html
+cgi.escape = html.escape
```

Comment:
It is better to avoid such lines, but if it is unavoidable, they should be after all imports